src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -64,19 +122,78 @@ export default class ReplyField extends React.Component<Props, State> {
         setCursorPosition(editorState.getSelection().getFocusOffset());
     };
 
+    handleMention = (): void => {
+        const { onMention } = this.props;
+        const { activeMention } = this.state;
+
+        if (onMention && activeMention) {
+            onMention(activeMention.mentionString);
+        }
+    };
+
+    getVirtualElement = (activeMention: Mention): VirtualElement | null => {
+        const selection = window.getSelection();
+        if (!selection?.focusNode) {
+            return null;
+        }
+
+        const range = selection?.getRangeAt(0);
+        const textNode = range.endContainer;
```

Comment:
I'm surprised this doesn't throw a type error, since it looks like we could be selecting a property on an undefined object. We should be judicious with our use of optional chaining and make sure we're handling the undefined cases properly.

Reply:
> I'm surprised this doesn't throw a type error, since it looks like we could be selecting a property on an undefined object. We should be judicious with our use of optional chaining and make sure we're handling the undefined cases properly.

It's actually very clever. In the above line, I've checked selection is not `null`, so the `?` here is not necessary, which is the reason that it doesn't throw an error.